### PR TITLE
doc: Modified README.md and ko/README.md

### PR DESCRIPTION
**Pull request by kangtegong**
Since Lua 5.1 script is now available and
dynamic tracing for AArch64 (ARM64) is possible in uftrace,
this commit updates both English README and Korean README
as containing those information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ For recording, the executable needs to be compiled with the `-pg`
 (or `-finstrument-functions`) option which generates profiling code
 (calling mcount or __cyg_profile_func_enter/exit) for each function.
 
-Note that, there's an experimental support for dynamic tracing on x86_64
+Note that, there's an experimental support for dynamic tracing on x86_64 and AArch64 (ARM64)
```
Comment (honggyukim) [inline README.md]: Please do not go over 80 characters in a single line.

File: doc/ko/README.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ record 명령어로 기록을 하려면, 각 함수마다 mcount 나 __cyg_profi
 호출하는 프로파일링 코드를 생성하기 위해 실행 파일을 `-pg`(혹은 `-finstrument-functions`) 옵션으로
 컴파일해야 한다.
 
-x86_64 아키텍처에서 (재)컴파일 과정이 필요하지 않은 동적 추적 기능이
+x86_64 와 AArch64 (ARM64)아키텍처에서 (재)컴파일 과정이 필요하지 않은 동적 추적 기능이
```
Comment (honggyukim) [inline doc/ko/README.md]: Please change it as follows:
```suggestion
x86_64 와 AArch64(ARM64) 아키텍처에서 (재)컴파일 과정이 필요하지 않은 동적 추적 기능이
```

**Comment by honggyukim** [comment]
Please change the commit message from AArch53 to AArch64.

**Comment by kangtegong** [comment]
amend commit message from AArch53 to AArch64
and 
modified line breaks so that there's less than 80 words in one line

**Comment by kangtegong** [comment]
> AArch53?



> AArch53?

My mistake..! I amended the commit message

File: README.md [hunk 3]
```diff
@@ -88,11 +88,12 @@ For recording, the executable needs to be compiled with the `-pg`
 (or `-finstrument-functions`) option which generates profiling code
 (calling mcount or __cyg_profile_func_enter/exit) for each function.
 
-Note that, there's an experimental support for dynamic tracing on x86_64
-which doesn't require such (re-)compilations.  Also recent compilers have
-some options to help uftrace to reduce tracing overhead with similar way
-(although it still needs recompilation of your program).  Please see
-[doc/uftrace-record.md](doc/uftrace-record.md) file.
+Note that, there's an experimental support for dynamic tracing on 
+x86_64 and AArch64 (ARM64) which doesn't require such (re-)compilations.
```
Comment (honggyukim) [inline README.md]: Please change "AArch64 (ARM64)" to "AArch64(ARM64)".

Comment (kangtegong) [reply README.md]: > Please change "AArch64 (ARM64)" to "AArch64(ARM64)".

Okay!

**Comment by kangtegong** [comment]
changed `AArch64 (ARM64)` to `AArch64(ARM64)`

File: README.md [hunk 4]
```diff
@@ -88,11 +88,12 @@ For recording, the executable needs to be compiled with the `-pg`
 (or `-finstrument-functions`) option which generates profiling code
 (calling mcount or __cyg_profile_func_enter/exit) for each function.
 
-Note that, there's an experimental support for dynamic tracing on x86_64
-which doesn't require such (re-)compilations.  Also recent compilers have
-some options to help uftrace to reduce tracing overhead with similar way
-(although it still needs recompilation of your program).  Please see
-[doc/uftrace-record.md](doc/uftrace-record.md) file.
+Note that, there's an experimental support for dynamic tracing on 
+x86_64 and AArch64(ARM64) which doesn't require such (re-)compilations.
```
Comment (honggyukim) [inline README.md]: Please change it as follows:
```suggestion
x86_64 and AArch64(ARM64) which do not require such (re-)compilations.
```

File: README.md [hunk 5]
```diff
@@ -88,11 +88,12 @@ For recording, the executable needs to be compiled with the `-pg`
...
-which doesn't require such (re-)compilations.  Also recent compilers have
-some options to help uftrace to reduce tracing overhead with similar way
-(although it still needs recompilation of your program).  Please see
-[doc/uftrace-record.md](doc/uftrace-record.md) file.
+Note that, there's an experimental support for dynamic tracing on 
+x86_64 and AArch64(ARM64) which doesn't require such (re-)compilations.  
+Also recent compilers have some options to help uftrace 
+to reduce tracing overhead with similar way
+(although it still needs recompilation of your program).  
+Please see [doc/uftrace-record.md](doc/uftrace-record.md) file.
```
Comment (honggyukim) [inline README.md]: Please remove all the white spaces at the end of lines. They are shown as red blocks in the diff output.
Screen Shot 2019-11-11 at 11 22 04 PM

**Comment by kangtegong** [comment]
removd blanks

File: README.md (hunk 4 above)
Comment (honggyukim) [reply README.md]: It's not changed yet.

Comment (kangtegong) [reply README.md]: Oh I forgot, I'll amend right away :)

File: README.md [hunk 6]
```diff
@@ -88,11 +88,12 @@ For recording, the executable needs to be compiled with the `-pg`
 (or `-finstrument-functions`) option which generates profiling code
 (calling mcount or __cyg_profile_func_enter/exit) for each function.
 
-Note that, there's an experimental support for dynamic tracing on x86_64
-which doesn't require such (re-)compilations.  Also recent compilers have
-some options to help uftrace to reduce tracing overhead with similar way
-(although it still needs recompilation of your program).  Please see
-[doc/uftrace-record.md](doc/uftrace-record.md) file.
+Note that, there's an experimental support for dynamic tracing on
+x86_64 and AArch64(ARM64) which do not require such (re-)compilations.
```
Comment (honggyukim) [inline README.md]: I'm sorry for asking you again. I was wrong. "which doesn't require" is actually correct.
Everything looks good except for it. Thanks.

Comment (kangtegong) [reply README.md]: > I'm sorry for asking you again. I was wrong. "which doesn't require" is actually correct.
> Everything looks good except for it. Thanks.

It's okay :) I'll fix it right away!

**Comment by kangtegong** [comment]
Fixed `which do not require` to `which doesn't require`

**Pull request merged**